config/application.rb

```diff
@@ -73,6 +73,8 @@ class Application < Rails::Application
     config.cache_store = :mem_cache_store, { compress: true,
                                              namespace: ENV['RAILS_HOST_NAME'] || 'hitobito' }
 
+    config.active_storage.variant_processor = :mini_magick
```

Comment:
We have to make sure minimagick is installed in the deployed docker images as well as in the development setup then

Reply:
We already use mini_magick for image processsing.